static/app/components/structuredEventData/index.tsx

```diff
@@ -16,22 +16,19 @@ export type StructedEventDataConfig = {
 };
 
 interface StructuredDataProps {
-  depth: number;
```

Comment:
depth is an internal prop, `<RecursiveStructuredData>` needs it but this public facing `<StructuredData>` should not.